templates/eap-xp1-openjdk11-image-stream.json

```diff
@@ -0,0 +1,135 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-eap-xp1-openjdk11-openshift",
+        "annotations": {
+            "description": "ImageStream definition for JBoss Enterprise Application Platform Expansion Pack with OpenJDK 11",
+            "openshift.io/provider-display-name": "Red Hat, Inc."
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap-xp1-openjdk11-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP XP with OpenJDK 11",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "1.0"
+                }
+            },
+            "labels": {
+                "xpaas": "1.0"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest released build of JBoss EAP XP",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eapxp:1.0,javaee:8,java:11",
```

Comment:
I'm not sure about that tag... I think it should be `eap-xp:1`... I'll double check

Reply:
I'll also check if we want to add a `microprofile:3.3`tag...

Reply:
Ok, I fixed the eap-xp ones and made them consistent between builder and runtime, let me know on the microprofile:3.3 one and I can add that.